indico/modules/designer/forms.py

```diff
@@ -20,3 +21,9 @@ class AddTemplateForm(IndicoForm):
     type = IndicoEnumSelectField(_('Template'), enum=TemplateType, default=TemplateType.poster)
     is_clonable = BooleanField(_('Allow cloning'), widget=SwitchWidget(), default=True,
                                description=_('Allow cloning this template in subcategories and events'))
+
+
+class CloneTemplateForm(IndicoForm):
+    category = CategoryField(_('Category'), [DataRequired()], require_event_creation_rights=True,
+                             warning_message=_('The selected category does not contain any events, only subcategories.'
+                                               ' Please make sure that you really want to clone a template there.'))
```

Comment:
I don't think this is necessary. Cloning a template into a category with no events makes perfect sense (since it's inherited into the subcategories). So I'd simply not show a warning at all when using the category field here.

Reply:
Okay, having no text makes it appear still with the styles so I added a `show_warning_message=False` instead. 
<img width="530" alt="Screenshot 2023-05-15 at 17 38 33" src="https://github.com/indico/indico/assets/54508387/8a768b35-8e0b-4d7b-b74e-c1ec67cbf700">